packages/rum-core/src/browser/performanceCollection.ts

```diff
@@ -337,17 +366,9 @@ function handleRumPerformanceEntries(
   configuration: RumConfiguration,
   entries: Array<PerformanceEntry | RumPerformanceEntry>
 ) {
-  const rumPerformanceEntries = entries.filter(
-    (entry) =>
-      entry.entryType === 'resource' ||
-      entry.entryType === 'navigation' ||
-      entry.entryType === 'paint' ||
-      entry.entryType === 'longtask' ||
-      entry.entryType === 'largest-contentful-paint' ||
-      entry.entryType === 'first-input' ||
-      entry.entryType === 'layout-shift' ||
-      entry.entryType === 'event'
-  ) as RumPerformanceEntry[]
+  const rumPerformanceEntries = entries.filter((entry): entry is RumPerformanceEntry =>
+    objectHasValue(RumPerformanceEntryType, entry)
+  )
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I wonder if we really need to keep filtering entries like this.

Reply:
Interesting 🤔. Do you remember why it was done in the first place?

Reply:
FMU, we still have this case to filter https://github.com/DataDog/browser-sdk/blob/fbe0200005f0581fab050d2f337a0e0dfe07d925/packages/rum-core/src/browser/performanceCollection.ts#L154-L155

Reply:
Ah right that's why :D Ok nevermind then. We can always change things if/when we work on my suggestion here: https://github.com/DataDog/browser-sdk/pull/2355#discussion_r1302738495